src/com/lftechnology/java/training/alina/basic/PrintDayInWord.java

```diff
@@ -0,0 +1,87 @@
+package com.lftechnology.java.training.alina.basic;
+
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/**
+ * Write a program called PrintDayInWord, which prints “Sunday”, “Monday”, ...
+ * “Saturday” if the int variable "day" is 0, 1, ..., 6, respectively.
+ * Otherwise, it shall print “Not a valid day”.
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ *
+ */
+public class PrintDayInWord {
+
+	private static final Logger LOGGER = Logger.getLogger(PrintDayInWord.class.getName());
+
+	public static void main(String[] args) {
+		Scanner scanner = null;
```

Comment:
the other way to close the resource is using try-with-resource feature. It will reduce the no. of lines and the code will be more readable.
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
http://www.mkyong.com/java/try-with-resources-example-in-jdk-7/